prog/main.go

```diff
@@ -134,6 +136,52 @@ type appFlags struct {
 	consulInf       string
 }
 
+type containerLabelFilterFlagArray []string
```

Comment:
> Why worry about splitting up the label? That's already handled elsewhere

Parsing the label when parsing the arguments lets us print an error when launching Scope. As a rule of thumb, the earlier the errors, the better.